src/drivers/gps/gps.cpp

```diff
@@ -697,6 +697,17 @@ GPS::run()
 		}
 	}
 
+	if (_instance == Instance::Main) {
+		handle = param_find("GPS_1_GNSS");
+
+	} else if (_instance == Instance::Secondary) {
+		handle = param_find("GPS_2_GNSS");
+	}
```

Comment:
Can you handle the `else` case here as well? Currently it would just use an unexpected handle (`GPS_UBX_MODE`).

Reply:
I handled this implicitly now by setting a default value and just reading the parameter when the handle was set.